examples/franka_example.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import cv2
+import numpy as np
+
+import habitat
+
+SENSOR_KEY = "rgb"
+
+
+def display_grayscale(image):
+    img_bgr = np.repeat(image, 3, 2)
+    cv2.imshow("Depth Sensor", img_bgr)
+    _ = cv2.waitKey(0)
+
+
+def display_rgb(image):
+    img_bgr = image[..., ::-1]
+    cv2.imshow("RGB", img_bgr)
+    _ = cv2.waitKey(0)
+
+
+def example():
+    # Note: Use with for the example testing, doesn't need to be like this on the README
+
+    with habitat.Env(
+        config=habitat.get_config("configs/tasks/franka_point.yaml")
+    ) as env:
+        print("Environment creation successful")
+        observations = env.reset()  # noqa: F841
+        display_rgb(observations[SENSOR_KEY])
```

Comment:
This script can be run from the pytest suite with a little more effort. The main thing missing is that there needs to be a way to disable displaying images so that it doesn't cause pytest to hang on headless machines (see test_examples.py for more details).